inc/change_project.class.php

```diff
@@ -70,6 +70,30 @@ static function getTypeName($nb = 0) {
 
 
    /**
+    * Duplicate all changes from a project template to his clone
+    *
+    * @since version 0.84
+    *
+    * @param $oldid
```

Comment:
Methods parameters must be documented as follows: `@param type $varname Some doc` (see http://glpi-developer-documentation.readthedocs.io/en/master/codingstandards.html#parameters-documentation)